gcages/tests/regression/test_end_to_end_ar6_regression.py

```diff
@@ -0,0 +1,149 @@
+"""
+Regression tests of raw emissions to post-processed output compared to AR6
+"""
+
+from __future__ import annotations
+
+import multiprocessing
+import platform
+from pathlib import Path
+
+import pandas as pd
+import pandas_indexing as pix
+import pytest
+
+from gcages.ar6 import run_ar6_workflow
+from gcages.database import GCDB
+from gcages.testing import (
+    AR6_IPS,
+    assert_frame_equal,
+    create_model_scenario_test_cases,
+    get_all_model_scenarios,
+    get_ar6_metadata_outputs,
+    get_ar6_raw_emissions,
+    get_ar6_temperature_outputs,
+)
+
+TEST_DATA_DIR = Path(__file__).parents[1] / "test-data"
+RUN_OUTPUT_DB_DIR = TEST_DATA_DIR / "end-to-end-running-ar6-regression"
+MODEL_SCENARIO_COMBOS_FILE = (
+    TEST_DATA_DIR / "ar6_scenarios_raw_model_scenario_combinations.csv"
+)
+
+
+scenario_cases = pytest.mark.parametrize(
+    "model, scenario",
+    create_model_scenario_test_cases(
+        get_all_model_scenarios(MODEL_SCENARIO_COMBOS_FILE)
+    ),
+)
+
+
+def run_checks(raw: pd.DataFrame) -> None:
+    if platform.system() == "Darwin":
+        if platform.processor() == "arm":
+            magicc_exe = TEST_DATA_DIR / "magicc-v7.5.3/bin/magicc-darwin-arm64"
+
+        else:
+            raise NotImplementedError(platform.processor())
+
+    else:
+        raise NotImplementedError(platform.system())
+
+    res = run_ar6_workflow(
+        input_emissions=raw,
+        magicc_exe_path=magicc_exe,
+        magicc_prob_distribution_path=(
+            TEST_DATA_DIR / "magicc-v7.5.3/configs/600-member.json"
+        ),
+        batch_size_scenarios=5,
+        scm_output_variables=("Surface Air Temperature Change",),
+        scm_results_db=GCDB(RUN_OUTPUT_DB_DIR),
+        n_processes=multiprocessing.cpu_count(),
+        run_checks=False,  # TODO: turn this back on
```

Comment:
Is this still here because of the shift away from AR6 with the emissions update?